src/vue/components/SocialSharingPopup/SocialSharingPopup.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <privacy-manager v-slot="{ loadPrivacyManagerModal }">
+    <div v-show="isVisible" class="social-sharing-popup__container">
+      <div class="social-sharing-popup__headline">
+        <slot name="headline">
+          Deine Datensicherheit bei der Nutzung der Teilen-Funktion
+        </slot>
+      </div>
+      <div class="social-sharing-popup__description">
+        <slot name="description">
+          Um diesen Artikel oder andere Inhalte über Soziale Netzwerke zu teilen, brauchen wir deine Zustimmung für
+        </slot>
+        <a href="#" rel="noopener" target="_blank" @click.prevent="loadPrivacyManagerModal(privacyManagerId)">
+          diesen Zweck der Datenverarbeitung
+        </a>
```

Comment:
Dieses Element vielleicht auch noch in den description-Slot verschieben?

Reply:
Ich hatte ursprünglich gedacht, nur den Bereich in einen Slot zu schreiben, der vermutlich austauschbar sein soll.  Aber ja, auch das kann man dort hineinschreiben.